examples/saas-dynamic-dashboards/frontend/app/Services/service.ts

```
@@ -0,0 +1,28 @@
+}
+
+export async function getTestsService() {
+    try {
+        let config = {
+            method: "POST",
+            url: "/api/getTests"
+        }
+        const res = await client(config)
+        return res
+    } catch (error) {
+        throw error
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)